ext/nnstreamer/tensor_filter/tensor_filter_lua.cc

```diff
@@ -126,47 +215,121 @@ lua_subplugin::configure_instance (const GstTensorFilterProperties *prop)
     lua_close (L);
   }
 
-  L = luaL_newstate();
+  /** @todo scale for num_tensors > 1 */
+  inputInfo.num_tensors = 1;
+  outputInfo.num_tensors = 1;
+
+  L = lua_open ();
   luaL_openlibs (L);
 
-  script = loadScript (prop);
-  int load_stat = luaL_loadbuffer (L, script, strlen (script), script);
-  if (load_stat != 0) {
-    /** @todo Error handling with load_stat */
+  /** @todo Support script given by raw string */
+  // script = loadScript (prop);
+  // int load_stat = luaL_loadbuffer (L, script, strlen (script), script);
+  // if (load_stat != 0) {
+  //   /** @todo Error handling with load_stat */
+  //  return;
+  // }
+
+  if (!g_file_test (prop->model_files[0], G_FILE_TEST_EXISTS)) {
```

Comment:
Discussion:
GstTensorFilterFrameworkInfo.verify_model_path 1 to check the file in tensor_filter main,
then remove this block.

Reply:
When the lua script is given by model property as raw string, we should not take it as model path.

I just added the commit for supporting raw string script. If the given model prop value is invalid for file path, it takes it as lua script.

PTAL